src/components/icon/icon.tsx

```diff
@@ -101,4 +102,11 @@ export class Icon {
             container.innerHTML = svgData;
         }
     }
+
+    private renderImg(src: string) {
+        const container = this.host.shadowRoot.querySelector('div.container');
+        if (container) {
+            container.innerHTML = `<img src="${src}" />`;
```

Comment:
I don't think we need this hack for regular images, we should be able to just render the element as usual :smile: We had to do this for the icons to fix broken SVG etc.

Also, `img` tags "require" the `alt` attribute

Reply:
Yeah, I know, but it felt like the two might interfere with each other if I used different methods… 🤔… maybe they wouldn't? I'll have a look at it 👍

Reply:
Nah, it's being annoying. If you first render an icon, then add an src to render an image instead, the icon isn't removed. I'm sure that's solvable, but I think it's likely easier to just use the same hack for both.

Reply:
> `img` tags "require" the alt attribute

the `svg`s also require that to be honest.

Reply:
> > `img` tags "require" the `alt` attribute
> 
> the `svg`s also require that to be honest.

There's no `alt` attribute for the `svg` element in the spec